lib/charms/opensearch/v0/opensearch_base_charm.py

```diff
@@ -1535,6 +1542,14 @@ def _set_node_conf(self, nodes: List[Node]) -> None:
         else:
             computed_roles = ClusterTopology.generated_roles()
 
+        # If the failover orchestrator is the only data node in the cluster, remove the
+        # cluster-manager role from it to avoid it bootstrapping the cluster
```

Comment:
Could you add a comment about why we're doing it, something along the lines of: 
"as it is the duty of the main orchestrator which should then broadcast the security_index_initialized" flag in the peer cluster orchestrator relation"